code-collab-app/src/editorpage.js

```diff
@@ -47,36 +30,22 @@ class Editorpage extends React.Component {
     if (!cellContents) {
       cellContents = '';
     }
-
-    this.setState({
```

Comment:
Same as comment on line 29. Now we set the state in peer_data which comes back to this page by props.